### PR TITLE
add contributors for sept launch

### PR DESCRIPTION
**Pull request by johngruen**
Part of #1411

### CODE REVIEW

**Comment by johngruen** [comment]
@clouserw wanna merge this when it passes?

**Comment by ghost** [comment]
Kind of.  The last one is failing integration tests right now on a line that has a comment starting with "HACK" ;_;

File: content-src/experiments/min-vid.yaml [hunk 1]
```diff
@@ -48,8 +48,16 @@ notifications: []
 contributors:
   -
     display_name: 'David Justice'
-    title: 'Software Engineer'
-    avatar: /static/images/experiments/avatars/generic-profile.png
+    title: 'Engineer'
+    avatar: /static/images/experiments/avatars/dave-justice.jpg
+  -
+    display_name: 'David Justice'
+    title: 'Senior Engieer'
```
Comment (chuckharmston) [inline content-src/experiments/min-vid.yaml]: Typo

File: content-src/experiments/min-vid.yaml [hunk 2]
```diff
@@ -48,8 +48,16 @@ notifications: []
 contributors:
   -
     display_name: 'David Justice'
-    title: 'Software Engineer'
-    avatar: /static/images/experiments/avatars/generic-profile.png
+    title: 'Engineer'
+    avatar: /static/images/experiments/avatars/dave-justice.jpg
+  -
+    display_name: 'David Justice'
```
Comment (ghost) [inline content-src/experiments/min-vid.yaml]: I think this should be Jared

**Review by chuckharmston — APPROVED** [review]
One typo that I forgot to make a part of a review.

I'm going to approve this and let you self-merge it.

File: content-src/experiments/min-vid.yaml (hunk 1 above)
Comment (ghost) [reply content-src/experiments/min-vid.yaml]: If this is Jared, this is Staff Engineer

**Comment by johngruen** [comment]
good spot @chuckharmston, i'm pretty punchy at this point :)

**Review by ghost — APPROVED** [review]

**Pull request merged**